shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -558,6 +564,13 @@ - (BOOL)runWithEntrypoint:(NSString*)entrypoint {
   return [self runWithEntrypoint:entrypoint libraryURI:nil];
 }
 
+- (void)notifyLowMemory {
+  if (_shell) {
+    _shell->NotifyLowMemoryWarning();
+  }
+  [_systemChannel sendMessage:@{@"type" : @"memoryPressure"}];
```

Comment:
The current API is annoying in that implementation must remember to also send the message along the system channel. The [Embedder API ](https://github.com/flutter/engine/blob/2ff740e9f8b5f22319b83469ad899d1f3e376118/shell/platform/embedder/embedder.cc#L1833) has to do this too. Can we just make `Shell::NotifyLowMemoryWarning` send the platform message as well (and update the embedder API impl to account for the same)

Reply:
I think that can be done, but would prefer to do it as a separate patch.

Reply:
Sounds good.